### PR TITLE
Document hf.co webhook publicly

### PR DESCRIPTION
**Pull request by julien-c**
(PR on top of #155 as it's very related)

### CODE REVIEW

**Review by Pierrci — APPROVED** [review]

File: docs/hub/endpoints.md [hunk 1]
```diff
@@ -22,3 +22,20 @@ We have open endpoints that you can use to retrieve information from the Hub as
...
 /api/logout  <br/>                 POST                                          | Log out user.                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | `logout(token)`                        | ```headers = { authorization :  "Bearer $token" }```                                                                                     |
+
+## Webhook
+
+If you ever need to programmatically get notified about all changes/additions to models on the Hugging Face Hub, you can subscribe to the Hub webhook.
+
+When you're subscribed – meaning you sent us a URL on your side, that you want us to ping – we will call it over HTTP with the following payload:
+
+```python
+{ "add": "user/model_id" }
```
Comment (osanseviero) [inline docs/hub/endpoints.md:33]: Should we add one example with organization/model_id?

File: docs/hub/endpoints.md [hunk 2]
```diff
@@ -22,3 +22,20 @@ We have open endpoints that you can use to retrieve information from the Hub as
...
 /api/logout  <br/>                 POST                                          | Log out user.                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | `logout(token)`                        | ```headers = { authorization :  "Bearer $token" }```                                                                                     |
+
+## Webhook
+
+If you ever need to programmatically get notified about all changes/additions to models on the Hugging Face Hub, you can subscribe to the Hub webhook.
```
Comment (osanseviero) [inline docs/hub/endpoints.md:28]: Nits

```suggestion
If you ever need to programmatically get notified about all the changes/additions to model repositories on the Hugging Face Hub, you can subscribe to the Hugging Face Hub webhook.
```

File: docs/hub/endpoints.md [hunk 3]
```diff
@@ -22,3 +22,20 @@ We have open endpoints that you can use to retrieve information from the Hub as
...
+
+When you're subscribed – meaning you sent us a URL on your side, that you want us to ping – we will call it over HTTP with the following payload:
```
Comment (osanseviero) [inline docs/hub/endpoints.md:30]: More nits

```suggestion
When you're subscribed – meaning you sent us a URL on your side that you want us to ping – we will call it over HTTP with the following payload:
```

**Review by osanseviero — APPROVED** [review]
Looks great! This is very cool

File: docs/hub/endpoints.md [hunk 4]
```diff
@@ -22,3 +22,20 @@ We have open endpoints that you can use to retrieve information from the Hub as
...
+
+```python
+{ "add": "user/model_id" }
+# or
+{ "update": "user/model_id" }
```
Comment (julien-c) [inline docs/hub/endpoints.md:35]:
```suggestion
{ "update": "organization/model_id" }
```

**Pull request merged**